src/services/scsc.py

```
@@ -180,6 +178,25 @@ async def update_global_status(
                 pig.status = SCSCStatus.active
                 self.session.add(pig)
 
+            unprocessed_applicants = self.oldboy_repository.get_unprocessed()
+            for applicant in unprocessed_applicants:
+                try:
+                    await self.oldboy_service.process_applicant(applicant.id)
+                except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)